sdk/monitor/tests.yml

```diff
@@ -9,7 +9,7 @@ extends:
         - task: AzureCLI@2
           displayName: Set OIDC variables
           inputs:
-            azureSubscription: azure-sdk-tests
+            azureSubscription: azure-sdk-tests-public
```

Comment:
@benbp we still don't have a good solution for having different service connections inside of these presteps. For now I've just hardcoded public for this and identity. I also had to remove the UsGov and China clouds for the weekly here as those don't work today either.

Reply:
https://github.com/Azure/azure-sdk-tools/pull/9099 should resolve. I need to do repo-specific updates to plumb the right config through (since I'm not setting the vars by default, though we could consider doing that). I've done it for go (https://github.com/Azure/azure-sdk-for-go/pull/23549), python and net coming next.